tests/unit_tests/dft/include/compute_inplace.hpp

```diff
@@ -66,10 +83,24 @@ int DFT_Test<precision, domain>::test_in_place_buffer() {
                                        error_margin, std::cout));
     }
 
-    descriptor_t descriptor_back{ size };
+    descriptor_t descriptor_back = descriptor_factory<descriptor_t, dimms>::get(size);
     descriptor_back.set_value(oneapi::mkl::dft::config_param::PLACEMENT,
                               oneapi::mkl::dft::config_value::INPLACE);
-    descriptor_back.set_value(oneapi::mkl::dft::config_param::BACKWARD_SCALE, (1.0 / size));
+    descriptor_back.set_value(oneapi::mkl::dft::config_param::BACKWARD_SCALE, (1.0 / size_total));
+    if constexpr(domain == oneapi::mkl::dft::domain::REAL){
+        if constexpr(dimms == 2){
+            std::array<std::int64_t, dimms + 1> real_strides{0, size, 1};
+            std::array<std::int64_t, dimms + 1> complex_strides{0, size/2+1, 1};
```

Comment:
```suggestion
            std::array<std::int64_t, dimms + 1> real_strides{0, 2*(size/2 + 1), 1};
            std::array<std::int64_t, dimms + 1> complex_strides{0, (size/2 + 1), 1};
```
padding in real strides required for in-place transforms